internal/pkg/template/templates/environment/cf.yml

```diff
@@ -216,7 +260,7 @@ Resources:
     Type: AWS::ElasticLoadBalancingV2::LoadBalancer
     Properties:
       Scheme: internet-facing
-      SecurityGroups: [ !GetAtt PublicLoadBalancerSecurityGroup.GroupId ]
+      SecurityGroups: [ !GetAtt PublicHTTPLoadBalancerSecurityGroup.GroupId,  !GetAtt PublicHTTPSLoadBalancerSecurityGroup.GroupId]
```

Comment:
We are suddenly increasing the surface or requests that a load balancer accepts (allowing port 443 requests), is it possible to include this sec group only if HTTPS is required?

Reply:
It is not increasing since we have 443 open now no matter if it is https or not, but I do agree we should improve this behavior and make it optional :+1: